README.md

```diff
@@ -4,4 +4,4 @@
 
 ## installation
 
-`yarn add git@github.com:Mariana-Tek/eslint-config-mariana.git eslint-config-airbnb-base`
+`npm install git@github.com:Mariana-Tek/eslint-config-mariana.git eslint-config-airbnb-base`
```

Comment:
what are our dependencies vs. peerDependencies?

Reply:
Great question man. I think in our case, everything is goes in `peerDependencies`, based on this answer in [StackOverflow](https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies#22004559) and [these docs](https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#peerdependencies). Since our config is a plugin, which needs all of these packages as a sibling basically.

Installing `peerDependencies` is a bit different than installing other `dependencies` though, so we should update the readme to reflect that.

Reply:
I decided that using "peerDependencies" is overkill. My thought was that we don't want to necessarily lock down which version of eslint each project needs,  but we should be keeping this repo up-to-date and its not like any of this stuff gets into prod, so I decided to keep things simple.